extensions/tailwindcss/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "tailwindcss",
+  "title": "Tailwind CSS",
+  "description": "Utilities for Tailwind CSS",
```

Comment:
Considering that at the moment this extension is only about documentation, how about updating the title and description to be more specific.
title -> "Tailwind CSS Documentation"
description -> "Quickly search Tailwind CSS documentation and open it in the browser"?

Later you can change title to "Tailwind CSS" once you add more functionality